marketpulse/main/views.py

```diff
@@ -20,6 +22,16 @@ def home(request):
 def edit_fxosprice(request, username='', id=None):
     user = request.user
 
+    country_code = ''
```

Comment:
Please move this under ``if lat and lang``. You could also initialize ``country_code`` with None which will be converted in js ``null``.